packages/features/ee/managed-event-types/lib/handleChildrenEventTypes.ts

```diff
@@ -146,17 +143,6 @@ export default async function handleChildrenEventTypes({
   // Calculate if there are new workflows for which assigned members will get too
   const currentWorkflowIds = eventType.workflows?.map((wf) => wf.workflowId);
 
-  // Define hashedLink query input
-  const hashedLinkQuery = (userId: number) => {
-    return hashedLink
-      ? !connectedLink
-        ? { create: { link: generateHashedLink(userId) } }
-        : undefined
-      : connectedLink
-      ? { delete: true }
-      : undefined;
-  };
```

Comment:
This part is very confusing. I have completely removed it because (based on what I could understand):
- Earlier since we had only one hashedlink per eventType, in case team admin were allowing child event types to have hasedlinks, we would update child events to each have links when that settings was toggled or in case new users were added. Which means as soon as admin toggled it, we would create links and connect it for each child event.
- I don't think it makes sense to have it now because user can toggle it in child event and have the option to generate as many as they want.
- In case it was toggled, child events had links, and then the admin changed it to not allow, we would simply delete all links for each child.

**TLDR**: We don't create hashed links for child events now. Team admins only toggle the option and each user can go to their child event have the option to create them